Rectangle/zh-Hans.lproj/Main.strings

```diff
@@ -132,13 +132,13 @@
 "Eah-KL-kbn.title" = "变大";
 
 /* Class = "NSTextFieldCell"; title = "First Third"; ObjectID = "F12-EV-Lfz"; */
-"F12-EV-Lfz.title" = "左三分之一";
+"F12-EV-Lfz.title" = "左 1/3";
 
 /* Class = "NSMenu"; title = "Help"; ObjectID = "F2S-fz-NVQ"; */
 "F2S-fz-NVQ.title" = "帮助";
 
 /* Class = "NSTextFieldCell"; title = "Right Half"; ObjectID = "F8S-GI-LiB"; */
-"F8S-GI-LiB.title" = "右半";
+"F8S-GI-LiB.title" = "右";
```

Comment:
I don't know Chinese, but I'm assuming that the direct translation of "half" was left in there before so that it's set apart from the "Move Right", etc (without resizing) actions? If it makes more sense to drop the half from each of these actions for Chinese, that's fine with me - just figured I'd double check and make sure that's what we want to do.

Reply:
右 is right and 半 is half.
But when you combine these two characters into “右半”, it would be incorrect and strange, we don’t say that in Chinese, it would be like “very thank you” instead “thank you very much”.
The proper translation of right half is “右半边” meaning “right half side”, but this is unnecessary. 
So the “right” would be fine in Chinese. 
Hope you can understand 😊